packages/dialtone-icons/package.json

```diff
@@ -65,5 +64,6 @@
     "Icons",
     "Icon Library"
   ],
-  "license": "MIT"
+  "license": "MIT",
+  "sideEffects": false
```

Comment:
Can we explain or document what sideEffects are?

Reply:
This is a left-over for testing tree-shake on webpack.

I'm not sure this is working as expected, it's not well documented so I'll remove it as it does nothing right now.

Reply:
or maybe we do have to set it to false on all our projects or webpack will not tree shake them? kinda seems that way

Reply:
https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Reply:
Seems like dialtone-css needs to be sideEffects: true based on this, since it's imported globally and css-loader is used. https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free